content/docs/studio/user-guide/model-registry/add-a-model.md

```diff
@@ -1,9 +1,8 @@
 # Add a model
```

Comment:
@aguschin In point no. 4 below (`Enter the path of the model file as follows:`), 

        - First bullet: what if the model file is in remote but is tracked by dvc - which path to add here?
        - Second bullet: `repository path where the dvc reference to the model should be saved` How is this different from the path of `dvc.yaml`?

Reply:
> In the first bullet point, what if the model file is in remote but is tracked by dvc - which path to add here?

Path to the tracked file in the Git repo. Regardless of file location in remote, if the model was saved to `some/path` (inside the repo), you should enter `some/path`.

> In the second bullet point, repository path where the dvc reference to the model should be saved How is this different from the path of dvc.yaml?

Let's assume I have my model in `s3://bucket/file`. Studio BE will run `dvc import` (or `dvc import-url`, I'm not sure), creating a `.dvc` file inside the repo. The path for this file is what you're asking about. You can see it's shown in "add a model" once you specify remote path.

![Jun-27-2023 15-27-54](https://github.com/iterative/dvc.org/assets/6797716/bd7e4dd7-9c39-4c36-a05e-8149640106bf)

Reply:
Thanks @aguschin.
I've made a small change [here](https://github.com/iterative/dvc.org/pull/4638/commits/a00cd3e8ff07dc0b50bdeb0494ce6bba858251ea). Let me know if it doesn't make sense.

Reply:
LGTM!